Assets/LeapMotionModules/Interaction/InteractionHand.cs

```diff
@@ -120,14 +127,14 @@ private struct HoverCheckResults {
     }
     private HashSet<IInteractionBehaviour> _hoveredLastFrame = new HashSet<IInteractionBehaviour>();
     private HashSet<IInteractionBehaviour> _hoverableCache = new HashSet<IInteractionBehaviour>();
-    private IInteractionBehaviour[] tempPerFingerHovered = new IInteractionBehaviour[3];
+    private IInteractionBehaviour[] _tempPerFingerHovered = new IInteractionBehaviour[3];
```

Comment:
I still think it is strange that there is this single array field that is periodically re-assigned to the only existing results field.  I think they should be combined in some way.